Sample/Kitchen Sink/DemonstrationTools/UnitConverter.swift

```diff
@@ -0,0 +1,184 @@
+//
+//  Shape+Extensions.swift
+//  flight-ui-ios
+//
+//  Created by Appivate 2023
+//
+
+import Foundation
+import SwiftUI
+import FlightUI
+
+struct UnitConverter: View {
+    @EnvironmentObject var theme: Theme
+    @StateObject var themeManager = ThemeManager(current: .dark)
```

Comment:
The theme manager is only required when modifying the current theme (e.g. swapping from dark to light mode) so you actually just want to use the `@EnvironmentObject var theme: Theme` and then variables below such as the padding with `theme.padding.grid2x` etc 👍 

Might be one to add to the documentation! Realise now how it could be unclear haha 😅